package.json

```diff
@@ -50,7 +50,7 @@
     "@angular-builders/jest": "~7.1.2",
     "@angular-devkit/build-angular": "~0.11.4",
     "@angular/cli": "~7.1.4",
-    "@angular/compiler-cli": "~7.1.4",
+    "@angular/compiler-cli": "^7.2.12",
```

Comment:
These too - my apologies for not being clear on which changes I meant :(

Reply:
Its okay, I changed it as well as requested